visualizer/src/hello_world.cpp

```diff
@@ -1,11 +1,89 @@
 #include "AudioFile.h"
 #include<iostream>
+#include <string>
+#include <thread> 
+#include <chrono>
 
+#include "../out/_deps/sdl2-src/include/SDL.h"
+
+
+/*
+   Some helpers for abstracting away playing raw audio on two channels (X,Y)
+   Uses the SLD2 video/audio library to be cross-platform
+*/
+struct RawAudioPlayer {
+  /* Print error to cerr and exits code 1*/
+  private:
+    SDL_AudioSpec want, have;
```

Comment:
I'll refactor this out to it's own header file later I'm just dumping everything in one test file for now